Documentation/contributing/testing/ci.rst

```diff
@@ -212,11 +212,11 @@ illustrating which subset of tests the job runs.
 +----------------------------------------------------------------------------------------------------------------+-------------------+--------------------+
 | `Cilium-PR-Kubernetes-Upstream <https://jenkins.cilium.io/job/Cilium-PR-Kubernetes-Upstream/>`_                | test-upstream-k8s | No                 |
 +----------------------------------------------------------------------------------------------------------------+-------------------+--------------------+
-| `Cilium-PR-K8s-X.XX-kernel-4.9 <https://jenkins.cilium.io/view/PR/>`_ (where ``X.XX`` is a K8s version)        | test-X.XX-4.9     | No                 |
+| `Cilium-PR-K8s-X.XX-kernel-4.9 <https://jenkins.cilium.io/view/PR/>`_ (``X.XX`` = K8s version)                 | test-X.XX-4.9     | No                 |
 +----------------------------------------------------------------------------------------------------------------+-------------------+--------------------+
-| All non-required `Cilium-PR-K8s-X.XX-kernel-4.9 <https://jenkins.cilium.io/view/PR/>`_                         | test-older-k8s    | No                 |
+| All non-required `Cilium-PR-K8s-X.XX-kernel-4.9 <https://jenkins.cilium.io/view/PR/>`_                         | test-older-k8s    | Backports          |
 +----------------------------------------------------------------------------------------------------------------+-------------------+--------------------+
-| `Cilium-PR-Ginkgo-Tests-K8s <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-K8s/>`_                      | test-missed-k8s   | No                 |
+| `Cilium-PR-Ginkgo-Tests-K8s <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-K8s/>`_                      | test-missed-k8s   | Backports          |
```

Comment:
Actually this seems inaccurate: contrary to what I though, `Cilium-PR-Ginkgo-Tests-K8s` does not seem to be currently required on backport PRs. If anybody could confirm, I will remove this from the changes.

Reply:
I think it was broken for 1.9 branch, but otherwise we should require it to pass.

Reply:
So it should be required in all cases, not only for backports, right? So just `Yes` in this case.

Reply:
We don't require it for PRs against master branch.